data/admin/user.py

```diff
@@ -62,6 +62,15 @@ class MaCanteenUserAdmin(UserAdmin):
                 )
             },
         ),
+        (
+            _("Section pour les élu·e·s"),
+            {
+                "fields": (
+                    "is_elected",
+                    "departments",
```

Comment:
Is it possible to get the departments selection component to more resemble sectors rather than a long checklist ? It would be more ergonomic if it was searchable and didn't take so much space for this uncommon user type

Reply:
Added card in Jardin